### PR TITLE
Don't start admin node for replicated loglet cluster tests 

### PR DESCRIPTION
**Pull request by tillrohrmann**
The replicated loglet cluster tests assume that they are in control
of configuring the loglet. This only works if the cluster does not
start a `LogsController`. This is achieved by starting these tests w/o
Role::Admin. With the `LogController`, it happened that the log was
automatically reconfigured since the configured sequencer did not 
respond to heartbeats.

### CODE REVIEW

File: crates/local-cluster-runner/src/node/mod.rs [hunk 1]
```diff
@@ -143,32 +143,68 @@ impl Node {
...
+        Self::new_test_node_with_extra_node(
+            base_config,
+            binary_source,
+            enum_set!(Role::MetadataStore),
+            roles,
+            size,
+        )
+    }
+
+    fn new_test_node_with_extra_node(
```
Comment (pcholakov) [inline crates/local-cluster-runner/src/node/mod.rs:184]: Minor, I think this was meant to be:

```suggestion
    fn new_test_nodes_with_extra_node(
```

**Review by pcholakov — APPROVED** [review]
Nice! Thanks for fixing this @tillrohrmann

**Comment by AhmedSoliman** [comment]
I was about to ask actually. Can we have a configuration option to run admin without controller?

**Comment by tillrohrmann** [comment]
> I was about to ask actually. Can we have a configuration option to run admin without controller?

Good point. Will change it accordingly.

**Review by jackkleeman — APPROVED** [review]

**Pull request merged**